Flow.Launcher.Core/Flow.Launcher.Core.csproj

```diff
@@ -58,10 +58,6 @@
     <PackageReference Include="squirrel.windows" Version="1.5.2" />
   </ItemGroup>
   
-  <ItemGroup>
-    <Folder Include="Properties\" />
```

Comment:
is this removal intended? we still specify compile include at line 48 of the core csproj file

Reply:
Let me check. I don't know why this is removed.